README.md

```diff
@@ -0,0 +1,13 @@
+Tested on macos monterey
+
+## Have docker desktop installed
+
+To run container locally simply exec from your terminal:
+
+`./run_docker_conatiner_test.sh # after this you should be inside container`
```

Comment:
Yes I see typo here, but script is also named `./run_docker_conatiner_test.sh` so it will work